projects/canopy/src/lib/filter-buttons/filter-button/filter-button-select-multiple.stories.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
+import { FilterVariant } from '../filter-button.interface';
+
+@Component({
+  selector: 'lg-reactive-form-multiple',
+  template: `
+    <form [formGroup]="form">
+      <lg-filter-group [variant]="variant">
+        {{ label }}
+        <lg-filter-button formControlName="p60" [value]="true">P60</lg-filter-button>
+        <lg-filter-button formControlName="duplicateTax" [value]="true"
+          >Duplicate Tax Voucher
+        </lg-filter-button>
+        <lg-filter-button formControlName="payslips" [value]="true"
+          >Payslips</lg-filter-button
+        >
+        <lg-filter-button formControlName="policy" [value]="true"
+          >Policy Schedule
+        </lg-filter-button>
+      </lg-filter-group>
+    </form>
+  `,
+})
+export class ReactiveFormMultipleComponent {
+  @Input() label: string;
+
+  @Input() variant: FilterVariant = 'select-multiple';
+
+  @Input()
+  set disabled(disabled: boolean) {
+    if (disabled === true) {
+      this.form.controls.p60.disable();
+      this.form.controls.duplicateTax.disable();
+      this.form.controls.payslips.disable();
+      this.form.controls.policy.disable();
+    } else {
+      this.form.controls.p60.enable();
+      this.form.controls.duplicateTax.enable();
+      this.form.controls.payslips.enable();
+      this.form.controls.policy.enable();
+    }
+  }
+
+  get p60() {
+    return this.form.get('p60');
+  }
+  get duplicateTax() {
+    return this.form.get('duplicateTax');
+  }
+  get payslips() {
+    return this.form.get('payslips');
+  }
+  get policy() {
+    return this.form.get('policy');
+  }
+
+  @Output() typesChange: EventEmitter<void> = new EventEmitter();
+
+  form: FormGroup;
+
+  constructor(public fb: FormBuilder) {
+    this.form = this.fb.group({
+      p60: [{ value: '', disabled: false }],
```

Comment:
I wondered if we should make these a  bit more generic, to just 'option 1', 'option 2' etc. Or in the [past I have used brand colours as options](https://legal-and-general-canopy.netlify.app/?path=/story/components-form-radio--standard).

It's very minor, but might be nice if we have some form of consistency.

Reply:
Will change them to be more generic.

Reply:
Done.